extensions/amp-analytics/amp-analytics.md

```diff
@@ -375,7 +375,53 @@ The `visibilitySpec` is a set of conditions and properties that can be applied t
   - `waitFor` This property indicates that the visibility trigger should wait for a certain signal before tracking visibility. The supported values are     `none`, `ini-load` and `render-start`. If `waitFor` is undefined, it is defaulted to [`ini-load`](#initial-load-trigger) when selector is specified, or to `none` otherwise.
   - `continuousTimeMin` and `continuousTimeMax` These properties indicate that a request should be fired when (any part of) an element has been within the viewport for a continuous amount of time that is between the minimum and maximum specified times. The times are expressed in milliseconds. The `continuousTimeMin` is defaulted to 0 when not specified.
   - `totalTimeMin` and `totalTimeMax` These properties indicate that a request should be fired when (any part of) an element has been within the viewport for a total amount of time that is between the minimum and maximum specified times. The times are expressed in milliseconds. The `totalTimeMin` is defaulted to 0 when not specified.
-  - `visiblePercentageMin` and `visiblePercentageMax` These properties indicate that a request should be fired when the proportion of an element that is visible within the viewport is between the minimum and maximum specified percentages. Percentage values between 0 and 100 are valid. Note that the upper bound (`visiblePercentageMax`) is inclusive while the lower bound (`visiblePercentageMin`) is not. When these properties are defined along with other timing related properties, only the time when these properties are met are counted. They default to 0 and 100 when not specified.
+  - `visiblePercentageMin` and `visiblePercentageMax` These properties indicate that a request should be fired when the proportion of an element that is visible within the viewport is between the minimum and maximum specified percentages. Percentage values between 0 and 100 are valid. Note that the upper bound (`visiblePercentageMax`) is inclusive. The lower bound (`visiblePercentageMin`) is exclusive, unless both bounds are set to 0 or both are set to 100. If both bounds are set to 0, then the trigger will fire when the element is not visible. If both bounds are set to 100, then the trigger will fire when the element is fully visible. When these properties are defined along with other timing related properties, only the time when these properties are met are counted. They default to 0 and 100 when not specified.
+  - `repeat` If this property is true, then the trigger will fire each time the visibilitySpec conditions are met. For example, if `continuousTimeMin` is set to 1 second and the element remains visible for 3 seconds, then the trigger would fire once if `repeat` is false, but would fire 3 times if `repeat` is true. The value of `repeat` defaults to false if not specified.
```

Comment:
can we not use the duration conditions here as example for now? it's still a bit unclear what the expected behavior should be when there is `totalTimeMin` condition in the spec.

I suggest we gave example with 
```
visibilitySpec: {
  visiblePercentageMin: 50,
  repeat: true,
}
``` 

the expected behavior with duration condition spec is still up to discussion. we don't expect user to use it at the moment.

Reply:
Done